cfgov/v1/models/learn_page.py

```diff
@@ -117,6 +117,7 @@ class LearnPage(AbstractFilterPage):
         ('call_to_action', molecules.CallToAction()),
         ('email_signup', organisms.EmailSignUp()),
         ('video_player', organisms.VideoPlayer()),
+        ('audio_player', v1_blocks.AudioPlayer()),
```

Comment:
Can we also make the `AudioPlayer` block an option inside of full-width text blocks on learn pages, too (without the `.block` container)? Or maybe just inside of the full-width text block and not as a top level block? The latest mockups for the podcast page I've seen have the audio player surrounded by a heading and links that don't need to be separated from the audio player by our 60px block margins.

Reply:
I'm working on implementing an alternate approach that @niqjohnson and I discussed yesterday: including heading, body, and additional details fields in the audio player block.